visual-inspection/inspection-backend/inspection/model/model.py

```diff
@@ -64,7 +113,7 @@
     "sunglass": "ein Brennglas",
     "toaster": "ein Toaster",
     "toilet_seat": "eine Klobrille",
-    "velvet": "Samt",
+    "velvet": "measuring_cupSamt",
```

Comment:
```suggestion
    "velvet": "Samt",
```

Copy-and-paste issue?